msbuild/Xamarin.Localization.MSBuild/MSBStrings.resx

```diff
@@ -494,7 +494,7 @@
     </data>
     
     <data name="W0111" xml:space="preserve">
-        <value>The iCloud entitlement '{item.Key}' is not supported by the Provisioning Profile.
+        <value>The iCloud entitlement '{0}' is not supported by the Provisioning Profile.
```

Comment:
was that not fixed in a different PR (maybe not on `main`) ?

Reply:
Yes, I messed up when creating the PR.

Reply:
It's fixed now.